phasespace/fromdecay/genmultidecay.py

```diff
@@ -0,0 +1,290 @@
+import itertools
+from typing import Callable, Union
+
+import tensorflow as tf
+import tensorflow.experimental.numpy as tnp
+from particle import Particle
+
+from phasespace import GenParticle
+
+from .mass_functions import _DEFAULT_CONVERTER
+
+_MASS_WIDTH_TOLERANCE = 0.01

```

Comment:
Should these be made visible in case somebody wants/needs to change the values, in which case you should remove the underscore at the start of the names? Maybe not necessar since the constructor below allows for a tolerance to be set; fine.

Reply:
I agree that this should be made public, especially the DEFAULT_MASS_FUNC. Changed this in 0b6ec078e05213984811275d063028b07a477c1d